drake/solvers/MathematicalProgram.cpp

```diff
@@ -119,70 +129,68 @@ class NonlinearProgram : public MathematicalProgram {
 
 class LinearComplementarityProblem : public MathematicalProgram {
  public:
-  virtual bool solve(OptimizationProblem& prog) const override {
+  virtual bool Solve(OptimizationProblem& prog) const override {
     // TODO ggould given the Moby solver's meticulous use of temporaries, it
     // would be an easy performance win to reuse this solver object by making
     // a static place to store it.
     MobyLCPSolver solver;
-    return solver.solve(prog);
+    return solver.Solve(prog);
   }
-  virtual MathematicalProgramInterface*
-  addLinearComplementarityConstraint() override {
+  virtual MathematicalProgramInterface* AddLinearComplementarityConstraint()
+      override {
     return new LinearComplementarityProblem;
   };
 };
 
-class LeastSquares
-    : public NonlinearProgram {  // public LinearProgram, public
+class LeastSquares : public NonlinearProgram {  // public LinearProgram, public
```

Comment:
This comment is stupid, but I guess not your fault.
I will live with leaving it alone, and accept this montonically-better patch as-is.